src/common/colors.js

```diff
@@ -0,0 +1,63 @@
+import {
+  LIMITED_ACTION,
+  SOCIAL_DISTANCING,
+  SHELTER_IN_PLACE,
+  SHELTER_IN_PLACE_WORST_CASE,
+  PROJECTED,
+} from 'common/interventions';
```

Comment:
Should we instead `import { INTERVENTIONS } from 'common/interventions` and then use `INTERVENTIONS.LIMITED_ACTION` etc. in this file?